server/handlers.go

```diff
@@ -1,8 +1,19 @@
 package server
 
-import "github.com/gin-gonic/gin"
+import(
+	"github.com/gin-gonic/gin"
+
+	"github.com/parkmenow/PMN-api/data"
+	)
 
 // getHello defines the endpoint for initial test
 func getHello(c *gin.Context) {
 	c.String(200, "Hello World")
 }
+
+func userRegistration(c *gin.Context)  {
+	buf := make([]byte, 2048)
+	num, _ := c.Request.Body.Read(buf)
+	reqBody := string(buf[0:num])
```

Comment:
There is a better approach involving the use of gin context itself
`c.BindJSON(&user)`
This automatically binds the input json to the `var user` struct variable

Reply:
Cool! thanks!